src/rv2/ml_backends/keras_classification.py

```diff
@@ -0,0 +1,13 @@
+from rv2.core.ml_backend import MLBackend
+
+
+class KerasClassification(MLBackend):
```

Comment:
Are Backends many to one to Tasks? i.e. there wouldn't be a generic "Keras" backend that could do multiple tasks, that a Backend only can perform one task?

Reply:
Yes.

Reply:
Updating comments.